tools/topology/sof-tgl-max98357a-rt5682.m4

```diff
@@ -231,7 +231,7 @@ DAI_ADD(sof/pipe-dai-playback.m4,
 
 # PCM Low Latency, id 0
 dnl PCM_PLAYBACK_ADD(name, pcm_id, playback)
-PCM_PLAYBACK_ADD(max357a-spk, 0, PIPELINE_PCM_1)
+PCM_PLAYBACK_ADD(CODEC, 0, PIPELINE_PCM_1)
```

Comment:
this should be 'Speakers'

Reply:
@plbossart , Thanks for pointing out! I commit new one for your reviewing.